### DISCUSSION TITLE
Alert No Data Keep Last State

### DISCUSSION THREAD

**Discussion by Ftoy99** [grafana/grafana#71495] [Alerting]
This was a feature that was removed but was not brough back even though the issue was mentioned

https://github.com/grafana/grafana/issues/36376

The simple use case is when you are using prometheus for eg. Disk Usage

Lets say we have 90% disk usage and the state is "Alerting" if the node goes offline there for "No Data" the state will change to "Alerting", "No Data" or "OK" . When the node comes back online an email will be send again due to the 90% disk usage.

**Comment by texige01** [comment]
I am currently in need of this feature as well and cannot find a workaround

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82585. Feel free to continue the discussion around this there. Thank you!
